test/autoload/device_backend/device_backend.egg-info/entry_points.txt

```diff
@@ -0,0 +1,2 @@
+[torch.backends]
+device_backend = backend_pkg:autoload
```

Comment:
in fact, this file should be ignored, and the reason it is here is to declear the entry point of the backend package